main.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.run
+end
```

Comment:
Hi @Dorcas126 , your project isn't running, so I can't check if your console app works similarly to the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).  I'd suggest you work on that so as to enable the reviewer to test your project and give feedback 😄

![image](https://github.com/Dorcas126/oop-school-library/assets/20537712/21d0f932-21e5-42fa-9c5a-58258b3957c8)